pyquil/simulation/tools.py

```diff
@@ -168,14 +168,13 @@ def permutation_arbitrary(qubit_inds: Sequence[int], n_qubits: int) -> Tuple[np.
     adjacent qubits.
 
     :param qubit_inds: Qubit indices in the order the gate is
-        applied to.
+           applied to.
     :param n_qubits: Number of qubits in system
     :return:
         perm - permutation matrix providing the desired qubit reordering
-        qubit_arr - new indexing of qubits presented in left to right
-            decreasing index order. Should be identical to passed 'args'.
+        qubit_arr - new indexing of qubits presented in left to right decreasing index order. Should be identical to passed 'args'.
```

Comment:
Should be identical to `qubit_inds`?

Reply:
Good catch. I'm not even sure if that would be true to say, so I'm just going to remove the second sentence.